### ISSUE TITLE
Can we write raw sql queries, like numtel:mysql package

### ISSUE THREAD

**Issue by SUVADEEP** [nodets/meteor-mysql#1]
**Can we write raw sql queries, like numtel:mysql package**

var liveDb = new LiveMysql(Meteor.settings.mysql);

Meteor.publish('allPlayers', function(){
  return liveDb.select(
    `SELECT * FROM players ORDER BY score DESC`,
    [ { table: 'players' } ]
  );
});

1)If not can we have your a package as well as numtel package in the same implementation?
2)I know you guys still working on it , but is there any chance to use it in production?

**Comment by kataras** [comment]
yes of course the option is deep but you can, using .connection.connection. query("your query string here", function(err, results, fields){})

**Comment by SUVADEEP** [comment]
Thanks for the support, its really helpful.
One more issue which we are facing , when trying to use numtel live query and nodets package both in one implementation,it seems there is a conflict , and both are not working from the same app. 
We can see that u have live query package too for npm currently.
Can you let me know, if I can just port  your live query package with  Npm.require (i.e. thinking if there are async issues)?
It would be really helpful...if you package it for us. Need to create a quick prototype , and we are quite new to Meteor as well a Node...but fortunately love it ,at this point I hate my JAVA dev tag most :)

**Comment by kataras** [comment]
Ahah , look the mysql-live-client package is also at the nodets repository, it has good docs (I think, although my english not the best). For client side I suggest you to use the gulp (build tool) with webpack or browserify. For mysql-live (it's the server-side of the package) you can port it with Npm.require it's a normal nodejs package, mysql-live is based on node-mysql-wrapper also has good docs. If your app contains real-time communication between your clients then it's your best (and only?) option( I created this package because I coudln't find anything which will suit my needs , my apps were based on 90% at mysql+ real time communication).  But before you use it I have to tell you something, this package is ok I used it for real apps it's working it's stable but I am not a nodejs developer anymore, I switched to golang the past month, I will solve issues and e.t.c but I will not spend to much time on this package anymore ( Iris web framework controls me now...)
